3rdparty/Logger/elmavenlogger.cpp

```diff
@@ -0,0 +1,63 @@
+#include "elmavenlogger.h"
+#include <iostream>
+
+
+ElMavenLogger* ElMavenLogger::_dlog = nullptr;
+
+ElMavenLogger::ElMavenLogger()
+{
+    try {
+        spdlog::set_async_mode(8192);
+        _logger = spdlog::rotating_logger_mt("ElMavLogger", "/home/rishabh/.elmaven/logs/elMavLogs", 1048576 * 5, 3);
+        _logger->set_level(spdlog::level::info);
+    }
+    catch(const spdlog::spdlog_ex& exp) {
+        
+        std::cout << "failed initializing the logger: " << exp.what() ;
+    }
+    
+
+}
+
+void ElMavenLogger::init()
```

Comment:
@rish9511 What is this for?

Reply:
@sahil21 
just for better readability. It initializes our logger class